data/json/recipes/other/materials.json

```
@@ -105,6 +105,46 @@
     "tools": [ [ [ "sheet_metal", -1 ], [ "wood_panel", -1 ] ] ],
     "components": [ [ [ "splinter", 10 ], [ "stick", 4 ], [ "2x4", 4 ], [ "withered", 20 ] ], [ [ "water", 4 ], [ "water_clean", 4 ] ] ]
   },
+  {
+    "result": "10l_bronze",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 10 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [ [ [ "5l_bronze", 2 ] ] ]
+  },
+  {
+    "result": "5l_bronze",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 5 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [ [ [ "1l_bronze", 5 ] ] ]
+  },
+  {
+    "result": "1l_bronze",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "autolearn": true,
+    "using": [ [ "forging_standard", 1 ], [ "bronzesmithing_tools", 1 ] ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "//": "There is a slight loss here, but rounding up would mean you could create more magically.",
+    "components": [ [ [ "scrap_bronze", 25 ] ] ]
```

```suggestion
    "//": "There is a slight loss here, but rounding up would mean you could create more magically.",
    "components": [ [ [ "scrap_bronze", 26 ] ] ]
```
You were literally generating mass, not losing it.